articles/reporting/signals-analytics.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Signals Analytics"
+description: "Learn about the detailed reporting and analytics provided about your sending."
+---
+
+One of the major benefits of SparkPost is the vast amount of reporting and analytics provided. With over forty different metrics about the entire lifecycle of your email transmissions, you can understand the success of your program, track key numbers, and find areas to improve your sending program.
+
+## Summary Report
+
+The default reporting view is the Summary report, as seen below. It provides you the ability to select from all metrics SparkPost tracks, adjust reporting date ranges, and filter by campaign, sending domain, recipient domain, and more. [Learn more](./summary) about the Summary report.
+
+![The summary report view in SparkPost](media/signals-analytics/summary-report.png)
+
+## Bounce, Rejections, Accepted, Delayed Reports
+
+Bounce, Rejections, Accepted, and Delayed reports in SparkPost provide the same date and filtering tools as the Summary report, with additional data and visualizations. Each report focuses on a different type of event and provides a breakdown into the different categories in the event, along with messages which provide information about why the event happened. For example, for the Bounce report, you'll see what reason mailbox providers are returning when they bounce your email.
+
+![](media/signals-analytics/bounce-report.png)
+
+
+## Health Score
+
+The Health Score is a predictive score to help you monitor your email health and identify problems before they become apparent in through traditional metrics. Use it to discover areas of your sending that are low-performing and get recommendations on how to improve. [Learn more](./health-score) about the Health Score.
```

Comment:
remove the word "in"  so that the 1st sentence reads "before they become apparent through traditional"